frontend/event/eventController.js

```diff
@@ -179,6 +176,20 @@
             }
         }
 
+        function formatPatch(patch) {
```

Comment:
Why format the JSON instead of configuring directly the object before the patch?